contributors/join-us/localization.md

```diff
@@ -207,3 +207,49 @@ To add a new WIP language:
 2. Select a component on the resulting page, then select **Tools > Start new translation**. 
 3. Select the languages you want to add, then **Start new translation**.
 4. Repeat step 2 for every component within the [https://translate.mattermost.com/projects/i18n-wip](https://translate.mattermost.com/projects/i18n-wip) project.
+
+## Setting up new translations on Weblate
+
+### Step 0 - Permissions
+
+Register an account and get admin permissions to the [Weblate server](https://translate.mattermost.com/).
+
+### Step 1 -  Create a new Project (optional)
+
+This step makes since if the project has its own repository (e.g. Calls) or if it's helpful to have a separate translation entry point. Other option is to simply skip this step and to add a new component to an existing project.
+
+**Note**
+
+Make sure to enable reviews for the project. Go to project settings, workflow tab and select enable reviews:
```

Comment:
This note may read better if both the note prefix and the note body are together on the same line.